test/full-trace/index.spec.js

```diff
@@ -1,5 +1,8 @@
 describe('Throwing error', () => {
     it('should throw', () => {
-        throw new Error('foo');
+        function x() {
+            throw new Error('foo');
+        }
+        x();
```

Comment:
Let's not change existing tests, that's a bad practice. Better create a separate test and add it.

Reply:
The test was breaking for some reason. I figured adding another stack frame wouldn't hurt.

Reply:
> The test was breaking for some reason

It is not failing in master branch. In any case, changing existing test cases is the last effort. Please revert it back.

Reply:
@fabiosantoscode ^